cmake/Platform/Common/Configurations_common.cmake

```diff
@@ -44,6 +44,7 @@ ly_append_configurations_options(
     DEFINES_RELEASE
         _RELEASE
         RELEASE
+        AZ_RELEASE_BUILD=1
```

Comment:
Why `=1` ? The AZ_RELEASE_BUILD checks seem to only use '#ifdef' not '#if'

Reply:
Following the same pattern as `AZ_DEBUG_BUILD` and `AZ_PROFILE_BUILD` in the same file. But it's a good call. It should be fine to change it to not have `=1`, but I rather be cautious and keep the same pattern in this change and reduce impact and risk. They could all be changed at the same time in a subsequent PR.

Reply:
Yeah, that should be fine. In general, ```#if``` still evaluates to false if its not defined or defined as false.

Reply:
I did the change.

Reply:
Thanks! If all the checks already use #ifdef instead of #if, this is clearer.